yarn.lock

```diff
@@ -4969,6 +4969,11 @@ boolbase@^1.0.0, boolbase@~1.0.0:
   resolved "https://registry.yarnpkg.com/boolbase/-/boolbase-1.0.0.tgz#68dff5fbe60c51eb37725ea9e3ed310dcc1e776e"
   integrity sha1-aN/1++YMUes3cl6p4+0xDcwed24=
 
+bootstrap@^5.1.0:
```

Comment:
After removing bootstrap, please also reinstall packages/recreate the `yarn.lock` so that it doesn't appear in here again. Thank you!